source/common/network/application_protocol.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "envoy/stream_info/filter_state.h"
+
+namespace Envoy {
+namespace Network {
+
+/**
+ * ALPN to set in the upstream connection. Filters can use this one to override the ALPN in TLS
+ * context.
+ */
+class ApplicationProtocols : public StreamInfo::FilterState::Object {
+public:
+  ApplicationProtocols(const std::vector<std::string>& application_protocols)
```

Comment:
nit: explicit for constructor with one argument.

Reply:
Updated.